hazelcast-hibernate/hazelcast-hibernate3/src/test/java/com/hazelcast/hibernate/LocalRegionFactoryDefaultTest.java

```diff
@@ -60,7 +61,95 @@ protected Properties getCacheProperties() {
     }
 
     @Test
-    public void testEntity() {
+    public void testNonStrictReadWriteEntity() {
```

Comment:
No this is only one test. This test checks if an updated entity invalidates the cache properly. So you have to read the entities to have them cached. Then update one of them and re-read all to make sure that updated entity was invalidated. Because this test needed to be this long, an additional group of assertions seems useful to me to make sure that test was properly initialized. I think this way it will be easier to spot a problem in a failing test. The same applies with next two tests.